README.md

```diff
@@ -19,192 +90,345 @@ $ make
+-----
 
-We have to specify the following parameters for a [GridDB][1] foreign server:
+## CREATE SERVER options
 
-host : GridDB notification addres.  
-port : GridDB notification port  
-clustername : GridDB cluster name  
-database : GridDB database name (can be skipped if you want to connect `public` database) 
+`griddb_fdw` accepts the following options via the `CREATE SERVER` command:
 
-If you want to use fixed list mode instead of multicast mode, you have to specify `notification_member` instead of `host` and `port`:
-```
-CREATE SERVER griddb_svr FOREIGN DATA WRAPPER griddb_fdw OPTIONS(notification_member '10.18.18.164:10001',
-clustername 'griddbfdwTestCluster');
-```
+- **keep_connections** as *boolean*, optional
 
-### create user mapping
-```
-CREATE USER MAPPING FOR public SERVER griddb_svr OPTIONS(username 'admin', password 'testadmin');
-```
+  Controls whether griddb_fdw keeps the connections to the foreign server open so that the subsequent queries can re-use them. The default is on. If set to off, all connections to the foreign server will be discarded at the end of transaction. Losed connections will be re-established when they are necessary by future queries using a foreign table.
 
-We have to specify the following parameters for a user mapping:  
-username : GridDB username  
-password : GridDB password  
+- **host** as *string*, optional
 
-### import schema
-```
-IMPORT FOREIGN SCHEMA griddb_schema FROM SERVER griddb_svr INTO public;
-```
+  GridDB notification address.
 
-We can use the following parameters for import schema:  
-recreate : 'true' or 'false'. If 'true', table schema will be updated.  
+- **port** as *interger*, optional
 
-After schema is imported, we can access tables.
-To use CREATE FOREIGN TABLE is not recommended.
+  GridDB notification port.
+
+- **database** as *string*, optional
+
+  GridDB database name. Default is `public` if skipped.
+
+- **clustername** as *boolean*, optional
+
+  GridDB cluster name.
+  
+- **notification_member** as *string*, optional
+
+  If you want to use fixed list mode instead of multicast mode, you have to specify `notification_member` instead of `host` and `port`:
+`notification_member '10.18.18.18:10001'`
+
+- **updatable** as *boolean*, optional
+
+- **batch_size** as *integer*, optional
+
+## CREATE USER MAPPING options
+
+`griddb_fdw` accepts the following options via the `CREATE USER MAPPING`
+command:
+
+- **username**
+
+  The griddb username to connect as.
+
+- **password**
+
+  The griddb user's password.
+
+
+## CREATE FOREIGN TABLE options
+
+`griddb_fdw` accepts the following table-level options via the
+`CREATE FOREIGN TABLE` command.
+
+- **table_name** as *string*
+
+- **updatable** as *boolean*, optional
+
+- **use_remote_estimate** as *boolean*, optional
+
+- **batch_size** as *integer*, optional
+
+- **fdw_startup_cost** as *interger*, optional
+
+- **fdw_tuple_cost** as *integer*, optional
 
-### update and delete
+- **rowkey** as *string*, optional
```

Comment:
Could you remove this rowkey in this section as TABLE options?

Reply:
Ok. I think descriptions by authors for all options will be usefully. During studying `gridsb_fdw` I don't understand some options.

Reply:
Thank you for support.